flake.nix

```diff
@@ -18,11 +18,26 @@
       systems = [ "x86_64-linux" "aarch64-linux" "aarch64-darwin" "x86_64-darwin" ];
       perSystem = { pkgs, system, ... }:
         let
+          litani = pkgs.callPackage ./litani.nix { };
+          cbmc-viewer = pkgs.callPackage ./cbmc-viewer.nix { };
+          astyle = pkgs.astyle.overrideAttrs (old: rec {
+            version = "3.4.13";
+            src = pkgs.fetchurl {
+              url = "mirror://sourceforge/${old.pname}/${old.pname}-${version}.tar.bz2";
+              hash = "sha256-eKYQq9OelOD5E+nuXNoehbtizWM1U97LngDT2SAQGc4=";
+            };
+          });
           core = builtins.attrValues
             {
+              litani = litani; # 1.29.0
+              cbmc-viewer = cbmc-viewer; # 3.8
+              astyle = astyle;
+
               inherit (pkgs)
+                cbmc# 5.91.1
```

Comment:
I think we should do the work now to build `cbmc` from source, so we have an easier time updating it. E.g. `brew install cbmc` already gives you 5.95.

Reply:
After updating to nix channel 24.05, the version of cbmc should be `5.95.1` as well, so the comment here also be updated

Reply:
The version in nix is actually `5.95.1` - sorry about that. I corrected it now. 

Do you think we should still build from source?